src/measure_theory/set_integral.lean

```diff
@@ -484,6 +484,31 @@ end
 
 end normed_group
 
+section mono
+
+variables {μ : measure α} {f g : α → ℝ} {s : set α}
+  (hf : integrable_on f s μ) (hg : integrable_on g s μ)
+
+lemma set_integral_mono_ae_restrict (h : f ≤ᵐ[μ.restrict s] g) :
+∫ a in s, f a ∂μ ≤ ∫ a in s, g a ∂μ :=
+integral_mono_ae hf hg h
+
+lemma set_integral_mono_ae (h : f ≤ᵐ[μ] g) :
+∫ a in s, f a ∂μ ≤ ∫ a in s, g a ∂μ :=
+set_integral_mono_ae_restrict hf hg
+  (filter.eventually.filter_mono ((ae_mono measure.restrict_le_self)) h)
+
+lemma set_integral_mono_on (hs : measurable_set s) (h : ∀ x ∈ s, f x ≤ g x) :
+∫ a in s, f a ∂μ ≤ ∫ a in s, g a ∂μ :=
+set_integral_mono_ae_restrict hf hg
+  (by simp [hs, eventually_le, eventually_inf_principal, ae_of_all _ h])
+
+lemma set_integral_mono (h : f ≤ g) :
+∫ a in s, f a ∂μ ≤ ∫ a in s, g a ∂μ :=
```

Comment:
Possibly worth stating as:
```suggestion
lemma set_integral_mono : monotone (λ f, ∫ a in s, f a ∂μ) :=
```
Or even having both.

Reply:
I don't think it can be done since there are requirements on `f` and `g`, so they need to be introduced before